### PR TITLE
Call assertion events for sync tests

### PR DESCRIPTION
**Pull request by harryi3t**

### CODE REVIEW

File: lib/sandbox/pmapi-setup-runner.js [hunk 1]
```diff
@@ -1,24 +1,13 @@
...
-
+module.exports = function (pm, execution, onAssertion) {
     /**
      * @param  {String} name
      * @param  {Function} assert
      * @chainable
      */
     pm.test = function (name, assert) {
-        var eventArgs = getAssertionObject(name, false);
+        var eventArgs = execution.getAssertionObject(name, false);
```
Comment (shamasis) [inline lib/sandbox/pmapi-setup-runner.js]: this should stay somewhere in utils or other place and not in `execution`

File: lib/sandbox/execution.js [hunk 2]
```diff
@@ -79,6 +80,16 @@ _.assign(Execution.prototype, {
 
             return value.toJSON();
         });
+    },
+
+    getAssertionObject: function (name, skipped, failed) {
```
Comment (shamasis) [inline lib/sandbox/execution.js]: this should stay somewhere in utils or other place and not in `execution`

**Comment by shamasis** [comment]
1. add pm.test.index() getter
2. let getAssertionObject stay as is (before it was moved)
3. simply trigger assertion object like structure in legacy interface with TODO for recovering the debt.

**Comment by harryi3t** [comment]
Opening a new PR from scratch

**Pull request closed without merging**